includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -360,6 +367,33 @@ public static function add_admin_menu_new_invalid_url_count() {
 			return;
 		}
 
+		$new_validation_error_urls = get_transient( static::NEW_VALIDATION_ERROR_URLS_COUNT_TRANSIENT );
+
+		if ( false === $new_validation_error_urls ) {
+			$new_validation_error_urls = static::get_validation_error_urls_count();
+			set_transient( static::NEW_VALIDATION_ERROR_URLS_COUNT_TRANSIENT, $new_validation_error_urls, DAY_IN_SECONDS );
+		}
+
+		if ( 0 === $new_validation_error_urls ) {
```

Comment:
FYI: The strict equality here caused a regression since integers stored in transients when no persistent object cache is used will get cast to strings. See #3478.